pages/c/_cluster/apps/charts.vue

```diff
@@ -344,8 +362,9 @@ export default {
           :class="{'pull-left': true, 'repo': true, [r.color]: true}"
           @input="toggleRepo(r, $event)"
         />
-      </div>
 
+        <Checkbox v-model="showPrerelease" class="repo mt-3" :label="t('catalog.charts.showPreRelease')" />
```

Comment:
I don't really get the point of the checkbox here, unless a chart has ONLY prereleases it won't change the list of whole charts displayed.

Reply:
That's fair. If this isn't relevant often I think it makes sense to move the toggle to the prefs page.